README.md

```diff
@@ -1,43 +1,31 @@
 # Rancher Logging Examples
 
-This repository contains 2 example charts for testing your logging setup and a simple application for generating logs.
+This repository contains an example charts for testing your logging setup.
 
 ## Charts
 
-### Simple Application
-The `simple-application` chart will deploy 2 pods that will constantly out logs.  This chart is recommended for when you want to configure your own logging setup and need a simple application to collect logs from.
-
 ### Rancher Logging Example
-The `rancher-logging-example` is an end to end setup of an application, multiple configurations for log collection, and an elasticsearch cluster as an output.  This chart is recommended if you want to see a working example of the [rancher-logging](https://rancher.com/docs/rancher/v2.x/en/logging/v2.5/) chart.  
+The `rancher-logging-example` is an end to end setup of an application, multiple configurations for log collection, and a simple output.  This chart is recommended if you want to see a working example of the [rancher-logging](https://rancher.com/docs/rancher/v2.x/en/logging/v2.5/) chart.  
 
 This chart demonstrates 3 scenarios:
 
 1. `all-logs` collects all logs in the cluster
 2. `logs-by-label` collects all logs for pods with the label `rancher.logging.example/app: log-generator`
 3. `namespace-logs` collects all logs for the namespace this chart is deployed in.
 
-#### Resource Requests and Limits
-
-You can change `elasticsearch` resource requests and limits from the upstream chart by adding the following `helm install` arguments...
+#### Installation
+To install this chart run:
 
-```bash
-helm install <required-and-optional-args> \
-    --set elasticsearch.resources.requests.cpu="0" \
-    --set elasticsearch.resources.requests.memory="0" \
-    --set elasticsearch.resources.limits.cpu="0" \
-    --set elasticsearch.resources.limits.memory="0"
+```shell
+git clone https://github.com/paynejacob/rancher-logging-examples
+cd rancher-logging-examples
+make install
 ```
 
-#### Checking ElasticSearch
-
-You can if `rancher-logging` is working by forwarding port `9200` and accessing the instance via `localhost`.
-
-```bash
-kubectl port-forward -n logging-example svc/elasticsearch-master 9200:9200
-```
 
-Now, navigate to your browser and access `http://localhost:9200/_cat/indices`.
-You should see three indices from the Flow(s), Output(s), ClusterFlow(s), and ClusterOuput(s) created.
+#### Checking the output
+ 
+To see information about the logs being collected you will need to proxy to the log output service.  The helm notes will include instructions for setting this up after a successful installation. Then you can view your logs by going to [http://localhost:8080](http://localhost:8080).  It can take a few seconds for logs to show up.

```

Comment:
nit:
"In order to view information about the logs being collected, you will need a proxy for the log output service"